.github/workflows/update.yml

```diff
@@ -54,8 +54,31 @@ jobs:
           $space/.venv3/bin/pip install -U jinja2==3.0.1
           $space/.venv3/bin/pip install https://github.com/ome/zeroc-ice-ubuntu2004/releases/download/0.2.0/zeroc_ice-3.6.5-cp38-cp38-linux_x86_64.whl
           WORKSPACE=$space BUILD=false SUFFIX=/dist bash autogen_omero.sh
+      - name: Find modified files
+        id: upstream_repo
+        run: |
+          files=$(git diff --name-only)
+          value="Changes from "
+          if [[ "openmicroscopy" =~ .*"$file".* ]]; then
+              value="${value} openmicroscopy"
+          elif [[ "omero-install" =~ .*"$file".* ]]; then
+              value="${value} omero-install"
+          elif [[ "omeroweb-install" =~ .*"$file".* ]]; then
+              value="${value} omeroweb-install"
+          fi
+          echo "::set-output name=value::$value"
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v4
         with:
           labels: include
           token: ${{secrets.ACTION_API_TOKEN}}
+          commit-message: ${{ steps.upstream_repo.outputs.value }}
+          delete-branch: true
+          title: 'Changes from upstream repositories'
```

Comment:
Is it possible that multiple repositories create one or could `${value}` be used here as well?

Reply:
Value will indicate the repositories that have been modified e.g. ``openmicroscopy, omero-install``

Reply:
Sorry, I noticed I was unclear. I mostly meant for `title`. So you're thinking is since there are multiple, we don't try to encode them into the title?

Reply:
HAppy to change the title

Reply:
https://github.com/jburel/ome-documentation/pull/16